tmt/steps/prepare/install.py

```diff
@@ -300,31 +362,31 @@ def sort_packages(self) -> None:
         self.required_packages = []
         for package in self.repository_packages:
             try:
-                output = self.guest.execute(
-                    ShellScript(f"rpm -q --whatprovides '{package}'"),
-                    silent=True
-                    )
+                output = self.guest.execute(Command('rpm', '-q', package), silent=True)
                 assert output.stdout
                 self.debug(f"Package '{output.stdout.strip()}' already installed.")
             except tmt.utils.RunError:
-                if self.skip:
+                if self.skip_missing:
                     self.recommended_packages.append(package)
                 else:
                     self.required_packages.append(package)
 
-    def prepare_command(self) -> None:
+    def prepare_command(self) -> Tuple[Command, Command]:
         """ Prepare installation command for rpm-ostree"""
-        # FIXME: cast() - https://github.com/teemtee/tmt/issues/1372
-        missing = cast(tmt.steps.prepare.PreparePlugin, self.parent).get("missing")
-        self.skip = True if missing == 'skip' else False
-        self.command = f"{self.sudo}rpm-ostree"
-        self.options = '--apply-live --idempotent --allow-inactive'
+
+        command = Command()
+
+        if self.use_sudo:
+            command += Command('rpm-ostree')
```

Comment:
Isn't `sudo` missing here?

Reply:
A mistake, this line should happen in any case, and `sudo` is not added. Fixed in [290ca3a](https://github.com/teemtee/tmt/pull/1772/commits/290ca3aac814455b67243c83743002f819fce975)

Reply:
Good, thanks.